lib/Gocdb_Services/User.php

```diff
@@ -514,6 +521,9 @@ protected function addUserPropertyLogic(\User $user, array $propArr) {
 
         // $this->addUserPropertyValidation($keyName, $keyValue);
 
+        // Set certificate DN to unique ID for now
+        $user->setCertificateDn($user->getId());
```

Comment:
This is a good candidate to squash away, as it later changes to `$user->setCertificateDn(null);`

Reply:
Do you still want the rest of the changes to registration as a separate commit, or can I squash that into the login changes?

Reply:
> Do you still want the rest of the changes to registration as a separate commit,

Yes please.